docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.8"
+networks:
+  intergration-test:
+    driver: bridge
+
+services:
+  db:
+    image: postgis/postgis
+    container_name: postgres
+    restart: on-failure
+    healthcheck:
+      test: /usr/bin/pg_isready
+      interval: 5s
+      timeout: 10s
+      retries: 120
+    environment:
+      POSTGRES_PASSWORD: mysecretpassword
+    ports:
+      - "5432:5432"
```

Comment:
please remove the port binding on the host if non needed. The end user may have a port clashin and postgresql is not secured

Reply:
port mapping was removed in https://github.com/geosolutions-it/MapStore2/pull/6490/commits/c842a9ca18f196332990cbae86921c130de76141